index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Professional Portfolio GERSON SANCHEZ</title>
+
+</head>
+
+<body>
+
+    <div class="container">
+        <header>
+            <nav class="navigation">
+                <p>John Doe</p>
```

Comment:
[OPTIONAL] I suggest making this logo a link element `<a>` instead of a paragraph. 

- The logo (if the design has one) is on the left and has a link (a tag).
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.